src/pages/studyView/StudyViewConfig.ts

```diff
@@ -83,6 +85,8 @@ export enum ChartTypeNameEnum {
     CNA_GENES_TABLE = 'table',
     GENOMIC_PROFILES_TABLE = 'table',
     CASE_LIST_TABLE = 'table',
+    SAMPLE_TREATMENTS_TABLE = 'table',
+    PATIENT_TREATMENTS_TABLE = 'table',
```

Comment:
Any difference between `SAMPLE_TREATMENTS_TABLE` and `PATIENT_TREATMENTS_TABLE`? If no much difference may be just create one type `TREATMENTS_TABLE`

Reply:
This enum is used in `onToggleOption`:
```TypeScript
         const chartTypeName =
                ChartTypeNameEnum[
                    this.props.store.chartsType.get(chartUniqueKey)!
                ];
```
By having separate enums that match with the names in `ChartTypeEnum`, you make the logic here much simpler. Unless I'm wrong about this, I'd rather keep it this way.